xarray/plot/plot.py

```diff
@@ -32,7 +32,9 @@ def _ensure_plottable(*args):
     Raise exception if there is anything in args that can't be plotted on
     an axis.
     """
-    plottypes = [np.floating, np.integer, np.timedelta64, np.datetime64]
+    from netCDF4 import datetime as nc4datetime
```

Comment:
This needs to be guarded behind a try/except clause, because netCDF4 is not a required dependency of xarray.  Use `from netcdftime import datetime` instead of netCDF4 (@jhamman is moving netcdftime to its own package). Also, put import statements at the top of the file.